### PR TITLE
cosmetic improvements

### PR DESCRIPTION
**Pull request by mcoughlin**
This PR makes a few cosmetic improvements.

### CODE REVIEW

File: static/js/components/AnnotationsTable.jsx [hunk 1]
```diff
@@ -224,15 +271,51 @@ const AnnotationsTable = ({ annotations, spectrumAnnotations = [] }) => {
     jumpToPage: false,
     pagination: true,
     tableBodyMaxHeight: "20rem",
+    customToolbar: () => (
+      <>
```
Comment (Theodlz) [inline static/js/components/AnnotationsTable.jsx:275]: I might be wrong but not sure these `<> </>` are needed here right?

File: static/js/components/AnnotationsTable.jsx [hunk 2]
```diff
@@ -75,6 +83,40 @@ const getMuiTheme = (theme) =>
     },
   });
 
+const dialogTitleStyles = (theme) => ({
+  root: {
```
Comment (Theodlz) [inline static/js/components/AnnotationsTable.jsx:87]: These isn't many annotations in the demo data to see how it renders when there is a lot of them. Did you try it by any chance while adding this styling? I am wondering if it doesn't overflow or anything, as it doesn't seem to be constrained in width and height.

File: static/js/components/CommentList.jsx [hunk 3]
```diff
@@ -306,6 +309,10 @@ const CommentList = ({
 
   comments = comments || [];
 
+  if (!includeBots) {
```
Comment (Theodlz) [inline static/js/components/CommentList.jsx:312]: Awesome! Thanks for adding that it's super nice.

**Review by Theodlz — APPROVED** [review]
Few minor comments, but thanks a lot for taking care of these so quickly.

File: skyportal/plot.py [hunk 4]
```diff
@@ -974,7 +974,7 @@ def mark_detections(plot, detection_dates, ymin, ymax):
     first_r = plot.line(
         x=np.full(5000, first),
         y=y,
-        line_alpha=0.5,
+        line_alpha=0.1,
```
[the remark this answers is not in the document]
Comment (mcoughlin) [reply skyportal/plot.py]: added.

File: static/js/components/AnnotationsTable.jsx (hunk 1 above)
Comment (mcoughlin) [reply static/js/components/AnnotationsTable.jsx]: good.

File: static/js/components/CommentList.jsx (hunk 3 above)
Comment (mcoughlin) [reply static/js/components/CommentList.jsx:312]: great.

File: static/js/components/AnnotationsTable.jsx (hunk 2 above)
Comment (mcoughlin) [reply static/js/components/AnnotationsTable.jsx:87]: I just tested it with what is in there. We don't have all that many columns, and the height seems OK. I suggest we move forward like this for now and then see how it works in production.

**Pull request merged**